mixins/form.jsx

```diff
@@ -193,7 +193,7 @@ module.exports = {
     if (errorType === "card_error" && cardError) {
       form.error(cardError.field, cardError.message);
     } else {
-      form.error("other", this.getIntlMessage('try_again_later'));
+      form.error("other", this.getIntlMessage('try_again_later') + " [" + errorType + "]");
```

Comment:
This is in response to an actual screenshot in an email we got, which we couldn't fix because we knew nothing about it.

I wouldn't have sent it either.